README.md

```diff
@@ -13,11 +13,12 @@ environment. More informations can be found at its own repo:
 - [Kubernetes module](modules/kubernetes/)
 - - [Module maintainers](MAINTAINERS.md)
 
-*NOTE*: It only suppport terraform 0.12.x onward. 
+*NOTE*: It only suppport terraform 0.13.x onward. 
 
```

Comment:
Just some spelling and things... *NOTE*: It only supports Terraform 0.13.x onward.

Reply:
Thanks, fixed.